### DISCUSSION TITLE
Multiple checkbox with same name

### DISCUSSION THREAD

**Discussion by sakjain02** [livewire/livewire#5657] [1. Help]
Hey folks,
Does anyone know why selecting only one checkbox marks all other checkbox selected.
Any alternate of wire:model.defer because only this helps me but I don't want to use it as submission value depends on another model request.

**Comment by joshhanley** [comment]
@sakjain02 I'm not exactly shore what you mean, can you share your checkboxes code so I can have a look?

**Reply by sakjain02** [reply]
I made it work though it was my bad . Thank You @joshhanley
